src/CONST.ts

```diff
@@ -5251,6 +5251,9 @@ const CONST = {
             DRAFTS: 'drafts',
             FINISHED: 'finished',
         },
+        TYPE: {
+            EXPENSE: 'expense',
+        },
```

Comment:
NAB we should eventually move this to `CONST.SEARCH.DATA_TYPES.EXPENSE` and get rid of transactions and reports. We'll do so as part of v2.2.